.gitpod.yml

```diff
@@ -1,5 +1,5 @@
 tasks:
-  - init: yarn install
+  - init: cd docs && yarn install
```

Comment:
This change is confusing to me because I don't see that you actually moved any files or changed the directory structure in this PR, you just changed the URL routing as far as I can tell.

Reply:
This was apparently not fixed elsewhere so I added the fix here